fonduer/parser/parser.py

```diff
@@ -288,100 +406,62 @@ def _parse_sentence(self, node, state):
             for (rgx, replace) in self.replacements:
                 text = rgx.sub(replace, text)
 
-            # Lingual Parse
-            document = state["document"]
-            for parts in self.lingual_parse(document, text):
-                (_, _, _, char_end) = split_stable_id(parts["stable_id"])
-                parts["document"] = document
-                parts["sentence_num"] = state["sentence"]["idx"]
-                abs_sentence_offset_end = (
-                    state["sentence"]["abs_offset"]
-                    + parts["char_offsets"][-1]
-                    + len(parts["words"][-1])
-                )
-                parts["stable_id"] = construct_stable_id(
-                    document,
-                    "sentence",
-                    state["sentence"]["abs_offset"],
-                    abs_sentence_offset_end,
+            # Process the Paragraph
+            stable_id = "{}::{}:{}".format(
+                state["document"].name, "paragraph", state["paragraph"]["idx"]
+            )
+            parts = {}
+            parts["stable_id"] = stable_id
+            parts["document"] = state["document"]
+            parts["position"] = state["paragraph"]["idx"]
+            parts["section"] = state["document"].sections[0]
```

Comment:
same here. why sections[0] here instead of the current section from parents?

Reply:
So to expand on this a little. Paragraph can be in Section, Table, or Cell. Would need to add several lines of logic to navigate up to the Section.